voice_assistant_pkg/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
We can probably remove the individual package LICENSE files and just keep a single one at the top level of the repository.

Reply:
done in barza/trailbot_interfaces PR